static/js/containers/VideoDetailPage_test.js

```diff
@@ -116,6 +116,12 @@ describe('VideoDetailPage', () => {
     assert.isTrue(wrapper.find("EditVideoFormDialog").exists());
   });
 
+  it('includes the delete button and dialog when the user has correct permissions', async () => {
```

Comment:
Maybe check the reverse of this too (delete button not present without admin permission).